cloudsim-plus/src/main/java/org/cloudbus/cloudsim/power/models/PowerModelHost.java

```diff
@@ -16,6 +16,28 @@
     public static final PowerModelHost NULL = new PowerModelHostNull();
 
     private Host host;
+    
+    /**
+     *  Power on delay of host
+     */
+    private double hostStartupDelay = 0;
+    
+
+    /**
+     *  Power off delay of host
+     */
+    private double hostShutDownDelay = 0;
```

Comment:
Since the attributes are inside the `PowerModelHost` class, the `host` name in `hostStartupDelay`, `hostShutDownDelay` and others is redundant.